cli/dcoscli/log.py

```diff
@@ -0,0 +1,197 @@
+import functools
+import sys
+import time
+
+from dcos import emitting, util
+from dcos.errors import DCOSException
+
+logger = util.get_logger(__name__)
+emitter = emitting.FlatEmitter()
+
+NO_FILE_EXCEPTION = DCOSException('No files exist. Exiting.')
+
+
+def log_files(mesos_files, follow, lines):
+    """Print the contents of the given `mesos_files`.  Behaves like unix
+    tail.
+
+    :param mesos_files: file objects to print
+    :type mesos_files: [MesosFile]
+    :param follow: same as unix tail's -f
+    :type follow: bool
+    :param lines: number of lines to print
+    :type lines: int
+    :rtype: None
+    """
+
+    fn = functools.partial(_read_last_lines, lines)
+    curr_header, mesos_files = _stream_files(None, fn, mesos_files)
+    if not mesos_files:
+        raise NO_FILE_EXCEPTION
```

Comment:
Is this correct? Should you raise a global/static object? What happens if you `raise NO_FILE_EXCEPTION` twice?